packages/react-ui-core/src/Ratings/RatingBar.js

```diff
@@ -41,10 +54,10 @@ export default class RatingBar extends PureComponent {
           classnames(
             theme.RatingBar_Item,
             theme[`RatingBar_Item-${index}`],
-            color && theme[`RatingBar_Item-${color}`]
+            this.props.fillColor && theme[`RatingBar_Item-${this.props.fillColor}`]
           )
         }
-        id={`rating-item-${index}`}
+        id={`${this.id}-rating-item-${index}`}
         width={`${this.fillWidth(index)}%`}
         {...props}
```

Comment:
`fillColor` is not being passed down into some of the other examples so is causing it to be added at this level as an invalid attribute.